actionview/test/template/tag_helper_test.rb

```diff
@@ -435,11 +435,12 @@ def test_data_attributes
 
   def test_aria_attributes
     ["aria", :aria].each { |aria|
-      assert_dom_equal '<a aria-a-float="3.14" aria-a-big-decimal="-123.456" aria-a-number="1" aria-array="[1,2,3]" aria-hash="{&quot;key&quot;:&quot;value&quot;}" aria-string-with-quotes="double&quot;quote&quot;party&quot;" aria-string="hello" aria-symbol="foo" />',
-        tag("a", aria => { a_float: 3.14, a_big_decimal: BigDecimal("-123.456"), a_number: 1, string: "hello", symbol: :foo, array: [1, 2, 3], hash: { key: "value" }, string_with_quotes: 'double"quote"party"' })
-      assert_dom_equal '<a aria-a-float="3.14" aria-a-big-decimal="-123.456" aria-a-number="1" aria-array="[1,2,3]" aria-hash="{&quot;key&quot;:&quot;value&quot;}" aria-string-with-quotes="double&quot;quote&quot;party&quot;" aria-string="hello" aria-symbol="foo" />',
-        tag.a(aria: { a_float: 3.14, a_big_decimal: BigDecimal("-123.456"), a_number: 1, string: "hello", symbol: :foo, array: [1, 2, 3], hash: { key: "value" }, string_with_quotes: 'double"quote"party"' })
+      assert_dom_equal '<a aria-a-float="3.14" aria-a-big-decimal="-123.456" aria-a-number="1" aria-array="1 2 3" aria-hash="truthy" aria-string-with-quotes="double&quot;quote&quot;party&quot;" aria-string="hello" aria-symbol="foo" />',
+        tag("a", aria => { a_float: 3.14, a_big_decimal: BigDecimal("-123.456"), a_number: 1, string: "hello", symbol: :foo, array: [1, 2, 3], hash: { truthy: true, falsey: false }, string_with_quotes: 'double"quote"party"' })
```

Comment:
@rafaelfranca I've updated this PR's implementation based on your feedback, but that also made me consider treating `aria-` attributes differently from `data-` attributes. To me, they're more similar to `class:`.

`aria-` prefixed Hashes don't seem common in practice, but it's probably more useful to pass them directly to `class_names` instead of serializing them to JSON.

Is that an improvement?

Reply:
I think I'd prefer to keep the current behavior unless we have a good reason to not do it.

Reply:
@rafaelfranca I don't think there are any aria- attributes that could make sense of an encoded JSON body. That being said, I think you're right -- that's probably something to consider with its own set of rationalizations. I'll open a separate PR for that behavior change so that we can keep this one focused on achieving the original desired behavior.